src/main/webapp/resources/js/components/samples/components/SequencingObjectList.tsx

```diff
@@ -34,32 +34,46 @@ import {
   fetchUpdatedSequencingObjects,
   useUpdateDefaultSampleSequencingObjectMutation,
 } from "../../../apis/samples/samples";
+import { CheckboxChangeEvent } from "antd/lib/checkbox";
+import { SampleSequencingObject, SequencingObject } from "../../../types/irida";
 
-const fileProcessTranslations = {
+const fileProcessTranslations: { [key: string]: any } = {
```

Comment:
See previous comment on dictionary definitions

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f